dashboard/test/ui/features/hamburger.feature

```diff
@@ -0,0 +1,177 @@
+    Given I am on "http://code.org/"
+    And I dismiss the language selector
+    And I create a student named "Sally Student"
+    Then I wait to see "#hamburger-icon"
+    And I click selector "#hamburger-icon"
+    Then I wait to see "#hamburger-contents"
+    And I see "#report-bug"
+    And I see "#support"
+    And I see ".divider#before-pegasus"
+    And I see "#learn"
+    And I see "#stats"
+    And I see "#help-us"
+    And I see "#about_entries"
+    And I see "#educate_entries"
+
+  Scenario: Teacher viewing hamburger dropdown (with expanded options) in English on desktop on Code.org
+    Given I am on "http://code.org/"
+    And I dismiss the language selector
+    And I create a teacher named "Tessa Teacher"
+    Then I wait to see "#hamburger-icon"
+    And I click selector "#hamburger-icon"
+    Then I wait to see "#hamburger-contents"
+    And I see "#report-bug"
+    And I see "#support"
+    And I see "#teacher-community"
+    And I see ".divider#before-pegasus"
+    And I see "#learn"
+    And I see "#stats"
+    And I see "#help-us"
+    And I see "#about_entries"
+    And I click selector "#about_entries"
+    And I wait to see "#about-us"
+    And I see "#educate_entries"
+    And I click selector "#educate_entries"
+    And I wait to see "#educate-overview"
+
+  Scenario: Signed out user viewing hamburger dropdown in Spanish on desktop on Code.org
+    Given I am on "http://code.org/lang/es"
+    Then I wait to see "#hamburger-icon"
+    And I click selector "#hamburger-icon"
+    Then I wait to see "#hamburger-contents"
+    And I see "#report-bug"
+    And I see "#support"
+    Then I wait until element "#teacher-community" is not visible
+    Then I wait until element "#learn" is not visible
+    Then I wait until element ".divider#before-pegasus" is not visible
+    Then I wait until element "#learn" is not visible
+    Then I wait until element "#educate-more" is not visible
+    Then I wait until element "#stats" is not visible
+    Then I wait until element "#help-us" is not visible
+    Given I am on "http://studio.code.org/reset_session/lang/en"
+    And I wait for 2 seconds
+
+  Scenario: Student viewing hamburger dropdown in Spanish on desktop on Code.org
+    Given I create a student named "Estrella Estudiante"
+    Then I am on "http://code.org/lang/es"
+    And I wait to see "#hamburger-icon"
+    And I click selector "#hamburger-icon"
+    Then I wait to see "#hamburger-contents"
+    And I see "#report-bug"
+    And I see "#support"
+    Then I wait until element "#teacher-community" is not visible
+    Then I wait until element "#learn" is not visible
+    Then I wait until element ".divider#before-pegasus" is not visible
+    Then I wait until element "#educate-more" is not visible
+    Then I wait until element "#stats" is not visible
+    Then I wait until element "#help-us" is not visible
+    Given I am on "http://studio.code.org/reset_session/lang/en"
+    And I wait for 2 seconds
```

Comment:
If changing language back is required (say, for the next scenario to work) then it might be more robust to make sure that we've successfully redirected to the next page after the previous navigation.